<?xml version="1.0"?>
<launch> 
  <arg name="world_file_path" default="/home/jan/git-projects/grasp/GraspIt-RosNode/graspit_ros/models/worlds/pal_gripper_world.xml" />
  <arg name="output_path" default="/home/jan/git-projects/grasp/GraspIt-RosNode/graspit_ros/RESULTS" />
  <node
    pkg="graspit_ros"
    type="graspit_ros_node_client"
    name="graspit_ros_node_client"
    args="$(arg world_file_path) $(arg output_path)"
    output = "screen"
  />
  <node
    pkg="graspit_ros"
    type="graspit_ros_node"
    name="graspit_ros_node" 
    output = "screen"
  />
</launch>




